CONTRIBUTING.md

```diff
@@ -85,24 +85,24 @@ Issues can be submitted by anyone - seasoned developers or newbies.
 8.  Submit a pull request.
 
     Within GitHub, visit this main repository and you should see a banner
-    suggesting to make a pull request. While you're writing up the pull
+    suggesting that you make a pull request. While you're writing up the pull
     request, you can add `Closes #XXX` in the message body where `#XXX` is the
-    issue you're fixing. So an example would be `Closes #42` would close issue
+    issue you're fixing. Therefore, an example would be `Closes #42` that would close the issue
```

Comment:
This becomes wordy and how it was before makes sense when reading the code-formatted text as special and not as you would with a normal word.

```suggestion
    issue you're fixing. Therefore, an example would be `Closes #42` would close issue
```

Reply:
Done.